src/connectionmanager.h

```diff
@@ -56,7 +56,7 @@ class ConnectionManager : public QObject, public LogSerializer {
     StateCheckSubscription,
     StateConnecting,
     StateConfirming,
-    StateOn,
+    StateIdle,
```

Comment:
StateIdle seems like a confusing name because it doesn't indicate that the VPN is on. Isn't the original StateOn name better?